docs/Sources/tuist/tuist.docc/Articles/Tuist/Users/sharing-code-across-manifests.md

```diff
@@ -0,0 +1,85 @@
+
+- It eases the **maintenance** because changes can be applied in one place and all the projects get the changes automatically.
+- It makes it possible to define **conventions** that new projects can conform to.
+- Projects are more **consistent** and therefore the likelihood of broken builds due inconsistencies is significantly less.
+- Adding a new projects becomes an easy task because we can reuse the existing logic.
+
+Reusing code across manifest files is possible in Tuist thanks to the concept of **project description helpers**.
+
+### Definition
+
+Project description helpers are Swift files that get compiled into a framework, `ProjectDescriptionHelpers`, that manifest files can import.
+
+> Structure:
+Tuist is not opinionated about the files structure and the content in them, so it's up to you to give them a structure that makes sense for your project.
+
+You can import them into your manifest file by adding an import statement at the top of the file:
+
+```swift
+// Project.swift
+import ProjectDescription
+import ProjectDescriptionHelpers
+```
+
+### Location
+
+Tuist traverses up the directory hierarchy until it finds a `Tuist` directory. Then it builds the helpers module including all the files under the `ProjectDescriptionHelpers` directory in the `Tuist` directory.
+
+### Example
+
+The snippets below contain an example of how we extend the `Project` model to add static constructors and how we use them from a `Project.swift` file:
+
+**Project+Templates.swift**
+
+```swift
+import ProjectDescription
+
+extension Project {
+  public static func featureFramework(name: String, dependencies: [TargetDependency] = []) -> Project {
+    return Project(
+        name: name,
+        targets: [
+            .target(
+                name: name,
+                destinations: .iOS,
+                product: .framework,
+                bundleId: "io.tuist.\(name)",
+                infoPlist: "\(name).plist",
+                sources: ["Sources/\(name)/**"],
+                resources: ["Resources/\(name)/**",],
+                dependencies: dependencies
+            ),
+            .target(
+                name: "\(name)Tests",
+                destinations: .iOS,
+                product: .unitTests,
+                bundleId: "io.tuist.\(name)Tests",
+                infoPlist: "\(name)Tests.plist",
+                sources: ["Sources/\(name)Tests/**"],
+                resources: ["Resources/\(name)Tests/**",],
+                dependencies: [.target(name: name)]
+            )
+        ]
+    )
+  }
+}
+```
+
+**Project.swift**
+
+```swift {2}
+import ProjectDescription
+import ProjectDescriptionHelpers
+
+let project = Project.featureFramework(name: "MyFeature")
+```
+
+> Tip: Note how through the function we are defining conventions about the name of the targets, the bundle identifier, and the folders structure.
```

Comment:
It'd be cool to eventually include a list of examples of how users are using helpers.